### PR TITLE
Add MC2

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: puppet/environments/seed_stack/manifests/site.pp [hunk 1]
```diff
@@ -12,6 +12,10 @@
     gluster_client_manage => false,
   }
 
+  # Try hack some ordering stuff to avoid mesos-slave dying so fast that
+  # upstart buries it.
+  Service['docker'] -> Service['mesos-slave']
```
Comment (jerith) [inline puppet/environments/seed_stack/manifests/site.pp]: This really belongs in `seed_stack::worker`, not here.

File: Vagrantfile [hunk 2]
```diff
@@ -81,7 +81,7 @@ Vagrant.configure(2) do |config|
   # Example for VirtualBox:
   #
   config.vm.provider "virtualbox" do |vb|
-    vb.memory = "1024"
+    vb.memory = "1536"
```
Comment (JayH5) [inline Vagrantfile]: You want to up this for controller/worker and combo instances?

Comment (jerith) [inline Vagrantfile]: We could get away with less on controller/worker than standalone, I suppose. I'll look up how to configure that.

**Comment by JayH5** [comment]
:+1:

**Pull request merged**